apps/web/playwright/organization/organization-creation.e2e.ts

```diff
@@ -103,26 +105,39 @@ test.describe("Organization", () => {
       await page.locator("button[type=submit]").click();
 
       // Waiting to be in next step URL
-      await page.waitForURL("/settings/organizations/*/onboard-admins");
+      await page.waitForURL("/settings/organizations/*/onboard-members");
     });
 
     await test.step("On-board administrators", async () => {
-      // Required field
-      await page.locator("button[type=submit]").click();
+      await page.waitForSelector('[data-testid="pending-member-list"]');
+      expect(await page.getByTestId("pending-member-item").count()).toBe(1);
 
-      // Happy path
       const adminEmail = users.trackEmail({ username: "rick", domain: `${orgDomain}.com` });
-      await page.locator('textarea[name="emails"]').fill(adminEmail);
-      await page.locator("button[type=submit]").click();
 
+      //can add members
+      await page.getByTestId("new-member-button").click();
+      await page.locator('[placeholder="email\\@example\\.com"]').fill(adminEmail);
+      await page.getByTestId("invite-new-member-button").click();
+      await expect(page.locator(`li:has-text("${adminEmail}")`)).toBeVisible();
       // Check if invited admin received the invitation email
       await expectInvitationEmailToBeReceived(
         page,
         emails,
         adminEmail,
         `${orgName}'s admin invited you to join the organization ${orgName} on Cal.com`
       );
+      await expect(page.getByTestId("pending-member-item")).toHaveCount(2);
+
+      // can remove members
+      await expect(page.getByTestId("pending-member-item")).toHaveCount(2);
+      const lastRemoveMemberButton = page.getByTestId("remove-member-button").last();
+      await lastRemoveMemberButton.click();
+      await page.waitForLoadState("networkidle");
+      await expect(page.getByTestId("pending-member-item")).toHaveCount(1);
+      // Cleanup here since this user is created without our fixtures.
+      await prisma.user.delete({ where: { email: adminEmail } });
```

Comment:
Not sure if we should delete this user.
 
This was also used in organization-invitation.e2e.ts

      const adminEmail = users.trackEmail({ username: "rick", domain: `${orgDomain}.com` });
      
      but no cleanup  code there

Reply:
> Not sure if we should delete this user.
> 
> This was also used in organization-invitation.e2e.ts
> 
> ```
>   const adminEmail = users.trackEmail({ username: "rick", domain: `${orgDomain}.com` });
>   
>   but no cleanup  code there
> ```

https://github.com/SomayChauhan/cal.com/blob/560288015c413f51fd7a2c7321b44beda4ce2443/apps/web/playwright/teams.e2e.ts#L340

copied it over from `Can create teams via Wizard` test